fix changing status during WP move/copy

NobodysNightmare opened the pull request:
## OpenProject work package

https://community.openproject.org/work_packages/5798
## Description

apparently the status_id param was set to expect an array of values while being moved in this commit: d7f399ab8db059153d2792f9db94450150c7ffdb
Probably because the spec started to expect that when it was moved to be an RSpec test.
I am not sure if this was ever a valid thing outside the tests...

NobodysNightmare committed:
actually allow the status to be set while moving

apparently the status_id param was set to expect an array of values while being moved in this commit: d7f399ab8db059153d2792f9db94450150c7ffdb
Probably because the spec started to expect that when it was moved to be an RSpec test.
I am not sure if this was ever a valid thing outside the tests...

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://nobodysnightmare-feature-change-status-during-move.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://nobodysnightmare-feature-change-status-during-move.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

NobodysNightmare committed:
fix moving of WPs without specifying a status

attributes without a value are actually represented by empty strings.
Thus only take the status_id if it is :present? (the inverse of :blank?)
